tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers.ts

```diff
@@ -410,6 +410,35 @@ const reducer = createReducer(
       };
     }
   ),
+  on(
+    actions.executionScrollToIndex,
+    (state: DebuggerState, action: {index: number}): DebuggerState => {
+      if (action.index < 0 || !Number.isInteger(action.index)) {
+        console.error(
+          `Attempt to scroll to negative or non-integer execution index ` +
+            `(${action.index})`
+        );
```

Comment:
If this cannot happen for any reasons (with UI), just throw an invariant error instead

Reply:
I made this just throw an `Error`. We don't have a dedicated class for invariant errors yet.